sdk/core/azure-core/src/test/java/com/azure/core/implementation/ResponseConstructorsCacheBenchMark.java

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.implementation;
+
+import com.azure.core.http.rest.Response;
+import com.azure.core.implementation.util.TypeUtil;
+import org.openjdk.jmh.Main;
```

Comment:
Would this work for all distributions of JDK? If not, and if it's only benchmarking, can we move this to a separate module?

Reply:
I had a quick chat with @karianna and @JonathanGiles on this.

JMH works best on OpenJDK or OpenJDK derivatives (could be enhanced for Eclipse OpenJ9 or Azul Zing benchmarking), but at this moment our sdks are targeting only OpenJDK variants. It’s unlikely that we’ll be using a non OpenJDK derivative on Azure.